content/en/docs/best-practices/channel-naming.md

```diff
@@ -175,9 +200,60 @@ candidate/fast/stable to denote the maturity level of the operator.  Using all
 as any channels you want to keep it in).
+
+# UPGRADES 
+
+OLM provides a variety of ways to specify updates between operator versions. Before
+continue in the following recommendations, make sure you understand its options by checking [Creating an update graph with OLM][upgrade-graph].
+
+## Recommended upgrade path
+
+Within a channel each patch release should be directly upgradable to the `head channel` from the latest version shipped
+in the previous minor version channel. Use skips or skipRange to provide this behaviour. (i.e. using the skipRange in the  
+`stable-v3.6` patch for `3.6.v<latest>` would mean setting the skipRange to be 
+`>= 3.5.<first z of the minor channel>. < 3.6.<new patch release>`, 
+where `3.5.z` represents the oldest `3.5.z` version for which you can provide direct upgrade support to `3.6.latest`)
+
+#### Example
+
+Let's imagine that you will publish the Operator bundle `3.6.30` (patch release under the channel `stable-v3.6`) and
+that you have published so far:
+
+- Operator bundles versions from `3.5.0` to `3.5.25` under the `stable-v3.5` channel
+- Operator bundles versions from `3.6.0` to `3.6.29` under the `stable-v3.6` channel
+
+Then, in this case, your newer patch release `3.6.30` would be configured with `skipRange: >=3.5.25 < 3.6.30` in
+order to only supports upgrading to the newest `3.6.z` from a pretty recent `3.5.z`. 
+
+You do not need necessarily to use `3.5.25`. Therefore, you should 
+use the oldest `3.5.z` version that supports upgrades from. Be aware that the ability to configure the path 
+using this example from an Operator version like `3.5.25` _(or `3.5.17`, or something else)_ 
+instead of `3.5.0` is a mechanism to constrain the support matrix when necessary, it's not the default recommendation.
+
+**Note** If you would like to ensure that the users of your Operator still able to install the Operator bundle version
+`3.6.29` by using the option `startingCSV` added manually in the subscription then, you will need to also use the `replaces: 3.6.29`
+to ensure the older Operator bundle does not get pruned from the index catalog via the OLM resolver.
+
+#### Attention (Be aware of the following example scenario)
+
+If you have the channel `stable-v3.7` where the head of channel is `3.7.10` and then, you provide a new patch release
+with a bug fix using the Operator bundle version `3.7.11` then, imagine that you configure your new publication to skip all
+that was published under the channel `stable-v3.7` (e.g. skipRange: `">= 3.7.0 < 3.7.11"` replaces: `3.6.z <latest release on 3.6 channel>` ).
+
+You could ensure that your users can move forward easier to the latest version since in this range `">= 3.7.0 < 3.7.11"`
+as they will be able to install the new `3.7.11` release from  `3.6.z <latest release on 3.6 channel>` directly 
+instead of get installed first the Operator version `3.7.0` which we know that have bug fixes in place already.
```

Comment:
```suggestion
instead of having to upgrade through Operator version `3.7.0` which may contain bugs that are already fixed in both the 3.6.z version they had installed, and the 3.7.11 version they are trying to get to.
```